src/test/datascience/notebook/interruptRestart.vscode.test.ts

```diff
@@ -206,7 +206,7 @@ suite('DataScience - VSCode Notebook - Restart/Interrupt/Cancel/Errors (slow)',
         commandManager.executeCommand(Commands.NotebookEditorInterruptKernel, vscEditor.document).then(noop, noop);
         // Stop the cell (cleaner way to tear down this test, else VS Code can hang due to the fact that we delete/close notebooks & rest of the code is trying to access it).
 
-        // IANHU
+        // KERNELPUSH
```

Comment:
Just to group with the other KERNELPUSH comments that I want to check into (I've marked the areas from the previous PR with this tag for investigation / follow up).